questionnaires/factories.py

```diff
@@ -0,0 +1,72 @@
+import factory
+from django.utils import timezone
+from factory.django import DjangoModelFactory
+from wagtail_factories import PageFactory
+
+from iogt_users.factories import UserFactory
```

Comment:
A lot of tests and factories depend on users. If we can't import user factories, how can we create test data?

Reply:
Is there a way to reference an abstract user instead, or to use `get_user_model` instead of referring to `iogt_users` directly?

Reply:
yes, we can use `get_user_model()` but it will significantly increase the amount of code to generate fixture data everywhere.

Let's do this when we address the HomePage factory issue.